### DISCUSSION TITLE
how to add a second pv string?

### DISCUSSION THREAD

**Discussion by giovanniincalza** [syssi/esphome-pipsolar#129] [Q&A]
Hy everyone, I'm using an esp8266 whit rs232-ttl converter connected to my axpert 8kw inverter. All is ok, but i can read the value of only one string. Can i configure a second string?

**Comment by syssi** [comment]
Could you try this YAML / feature branch instead: https://github.com/syssi/esphome-pipsolar/blob/pip8048/esp8266-example.yaml
